dist/profile/templates/kubernetes/resources/evergreen/secret.yaml.erb

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret 
+metadata: 
+    name: accountapp
```

Comment:
I think you forgot this resource :P 
all current data aren't needed and you should have the postgres_key